jetty-openid/src/main/config/etc/jetty-openid.xml

```diff
@@ -1,12 +1,24 @@
 <?xml version="1.0"?>
 <!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
 <Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Get id="ThreadPool" name="threadPool"/>
+  <New id="HttpClient" class="org.eclipse.jetty.client.HttpClient">
+    <Arg>
+      <New class="org.eclipse.jetty.util.ssl.SslContextFactory$Client">
+        <Set name="trustAll" type="boolean">false</Set>
+        <Set name="endpointIdentificationAlgorithm">https</Set>
+      </New>
+    </Arg>
+    <Set name="followRedirects" type="boolean">false</Set>
```

Comment:
This probably always works for token endpoint and metadata (though there's nothing in the spec that says a redirect isn't allowed). However, when we get to WebFinger, this won't work:

> A WebFinger resource MAY redirect the client; if it does, the
>   redirection MUST only be to an "https" URI and the client MUST
>   perform certificate validation again when redirected.

https://tools.ietf.org/html/rfc7033#section-4.2